.circleci/config.yml

```diff
@@ -50,6 +54,15 @@ commands:
           paths:
             - dist
 
+  # setup_bazel_rbe:
```

Comment:
Are we planning to do this still?

Reply:
After talking offline... it's not done today so we're not doing it in bazel.

Reply:
I think this was only required for e2e browser testing which is not on CI today so we're not doing it in bazel either.

Reply:
but the plan is to enable remote execution for build and testing right? I might be misunderstanding